Code/GraphMol/ForceFieldHelpers/FFConvenience.h

```diff
@@ -125,6 +126,22 @@ inline void OptimizeMoleculeConfs(ROMol &mol, ForceFields::ForceField &ff,
   }
 #endif
 }
+
+//! Convenience Function for generating an empty force Field with just
+/// the molecules' atoms position.
+/*
+  \param mol  The molecule which positions should be added to the force field.
+
+*/
+inline ForceFields::ForceField *constructEmptyForceField(ROMol &mol,
```

Comment:
In newer code we no longer write functions which return naked pointers.
Please return an `std::unique_ptr<Forcefield::Forcefield>` instead.

Reply:
I also think that a better name would be `createEmptyForceFieldForMol()`